<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>Input</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="INPUT_textEdit"/>
        </item>
       </layout>
      </widget>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="title">
        <string>Output</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QTextEdit" name="OUTPUT_textEdit">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="XML_Validate">
          <property name="text">
           <string>XML Validate</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Correct_XML_Button">
          <property name="text">
           <string>Correct XML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Minif_XML">
          <property name="text">
           <string>Minify XML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="XML_TO_JSON">
          <property name="text">
           <string>Convert XML to JSON</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Prettify">
          <property name="text">
           <string>Prettify</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="CompressButton">
          <property name="text">
           <string>Compress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Decompress_PushButton">
          <property name="text">
           <string>DeCompress</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="pushButton">
          <property name="text">
           <string>Post Search</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Graph_XML_Button">
          <property name="text">
           <string>Graph XML</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="Network_Analysis_Button">
          <property name="text">
           <string>Network Analysis</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="separator"/>
    <addaction name="actionSave_output_file"/>
    <addaction name="actionSave_as_Output_File"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionSelect_all"/>
    <addaction name="actionFind"/>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
    <widget class="QMenu" name="menuZoom_in">
     <property name="title">
      <string>Zoom in</string>
     </property>
     <addaction name="actionInput"/>
     <addaction name="actionOutput"/>
    </widget>
    <widget class="QMenu" name="menuZoom_out">
     <property name="title">
      <string>Zoom out</string>
     </property>
     <addaction name="actionInput_2"/>
     <addaction name="actionOutput_2"/>
    </widget>
    <addaction name="menuZoom_in"/>
    <addaction name="menuZoom_out"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuAbout_us_2">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionValidate_XML"/>
    <addaction name="actionMinify_XML"/>
    <addaction name="actionPrettify_XML"/>
    <addaction name="actionConvert_XML_to_JSON"/>
    <addaction name="actionCompress_XML_File"/>
    <addaction name="actionDecompress_XML_file"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionProject_Description"/>
    <addaction name="actionAbout_us"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuView"/>
   <addaction name="menuAbout_us_2"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="statusbar" stdset="0">
    <string/>
   </property>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save (Input File)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as (Input File)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+I</string>
   </property>
  </action>
  <action name="actionSave_output_file">
   <property name="text">
    <string>Save (Output File)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as_Output_File">
   <property name="text">
    <string>Save as (Output File)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
  <action name="actionSelect_all">
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionInput">
   <property name="text">
    <string>Input</string>
   </property>
  </action>
  <action name="actionOutput">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionInput_2">
   <property name="text">
    <string>Input</string>
   </property>
  </action>
  <action name="actionOutput_2">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionFind">
   <property name="text">
    <string>Find</string>
   </property>
  </action>
  <action name="actionProject_Description">
   <property name="text">
    <string>Project Description</string>
   </property>
  </action>
  <action name="actionAbout_us">
   <property name="text">
    <string>About us</string>
   </property>
  </action>
  <action name="actionInput_3">
   <property name="text">
    <string>Input</string>
   </property>
  </action>
  <action name="actionOutput_3">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionInput_4">
   <property name="text">
    <string>Input</string>
   </property>
  </action>
  <action name="actionOutput_4">
   <property name="text">
    <string>Output</string>
   </property>
  </action>
  <action name="actionValidate_XML">
   <property name="text">
    <string>Validate XML</string>
   </property>
  </action>
  <action name="actionMinify_XML">
   <property name="text">
    <string>Minify XML</string>
   </property>
  </action>
  <action name="actionPrettify_XML">
   <property name="text">
    <string>Prettify XML</string>
   </property>
  </action>
  <action name="actionConvert_XML_to_JSON">
   <property name="text">
    <string>Convert XML to JSON</string>
   </property>
  </action>
  <action name="actionCompress_XML_File">
   <property name="text">
    <string>Compress XML file</string>
   </property>
  </action>
  <action name="actionDecompress_XML_file">
   <property name="text">
    <string>Decompress XML file</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
